Package.swift

```diff
@@ -390,6 +408,96 @@ let package = Package(
       ]
     ),
 
+    .target(
+      name: "FirebaseFirestore",
+      dependencies: [
+        "FirebaseCore",
+        "leveldb",
+        .product(name: "nanopb", package: "nanopb"),
+        .product(name: "abseil", package: "abseil"),
+        .product(name: "gRPC-cpp", package: "gRPC"),
+      ],
+      path: "Firestore",
+      exclude: [
+        "Example/",
+        "core/test/",
+        "Protos/cpp/",
+        "Protos/protos/",
+        "Protos/Podfile",
+        "Swift/",
+        "fuzzing/",
+        "third_party/",
+        "CHANGELOG.md",
+        "README.md",
+        "Protos/CMakeLists.txt",
+        "CMakeLists.txt",
+        "Protos/lib/__init__.py",
+        "Protos/README.md",
+        "test.sh",
+        "Protos/build_protos.py",
+        "Protos/nanopb_cpp_generator.py",
+        "Protos/lib/pretty_printing.py",
+        "Source/CMakeLists.txt",
+        "core/CMakeLists.txt",
+        "core/src/util/config_detected.h.in",
+
+        // Exclude alternate implementations for other platforms
+        "core/src/api/input_validation_std.cc",
+        "core/src/remote/connectivity_monitor_noop.cc",
+        "core/src/util/filesystem_win.cc",
+        "core/src/util/hard_assert_stdio.cc",
+        "core/src/util/log_stdio.cc",
+        "core/src/util/secure_random_openssl.cc",
+      ],
+      sources: [
+        "Source/",
+        "Protos/nanopb/",
+        "Protos/objc/",
+        "core/include/",
+        "core/src",
+      ],
+      publicHeadersPath: "Source/Public",
+      cSettings: [
+        .headerSearchPath("../"),
+        .headerSearchPath("Source/Public"),
+        .headerSearchPath("Protos/nanopb"),
+        .headerSearchPath("Protos/objc/google/api"),
```

Comment:
This is obsolete: we no longer use Objective-C protocol buffers

Reply:
Removed here and in the podspec